eng/versioning/external_dependencies.txt

```diff
@@ -207,3 +228,7 @@ media_com.microsoft.azure:adal4j;1.2.0
 
 # sdk\storage\azure-storage-blob-cryptography\pom.xml
 storage_com.microsoft.azure:azure-storage;8.4.0
+
+
+# sdk\spring\azure-spring-boot\pom.xml
+spring_com.nimbusds:nimbus-jose-jwt;7.9
```

Comment:
Three comments:

1. I'm unclear why this is version 7.9. When I look at [spring-security-oauth2-jose 5.2.0.RELEASE](https://mvnrepository.com/artifact/org.springframework.security/spring-security-oauth2-jose/5.2.0.RELEASE), I see the version referenced there is 7.8. Should we not be keeping our dependencies in line with the spring dependencies?

2. I would rather that we make this special dependency (because of it's 'spring_' prefix) our 'official' dependency, and use that for all of our track two client libraries. The current 'official' dependency, as far as I can see, is only used in track one for microsoft-azure-eventhubs, so I would rather give that a special prefix instead // cc @JimSuplizio 

3. I would really like to have a comment associated with this dependency so that people know to not eagerly update the version and to instead move it in lockstep with the spring version we are supporting.

Reply:
We use version 7.9 because we include a `JWKSetCache` class which is not available in 7.8 version.

Reply:
Doesn't that potentially result in conflicts if we are bringing in 7.9 and spring relies on 7.8?

Reply:
Use `7.9` as the external dependency version for nimbus-jose-jwt, and also create an entry for track1 eventhubs library.